components/resource/texturemanager.cpp

```diff
@@ -194,6 +204,14 @@ namespace Resource
         }
     }
 
+#ifdef USE_TGA
+    inline bool TextureManager::ends_with(std::string const & value, std::string const & ending)
```

Comment:
Better yet, this shouldn't be a member function at all; it should be a static function in this cpp file. It doesn't rely on any members and it's not used outside of this file.